config/flags_internal.go

```diff
@@ -63,11 +70,30 @@ func parseFlags(cfg *Config, version string) {
 	}
 }
 
+func parseOwnerConsentCommandsFlag() []types.CommandType {
+	var listCommands string
+	flagSet := flag.NewFlagSet("", flag.ContinueOnError)
+	flagSet.SetOutput(io.Discard)
+	flagSet.StringVar(&listCommands, ownerConsentCommandsFlagID, EnvToString("OWNER_CONSENT_COMMANDS", ""), ownerConsentCommandsDesc)
+	if err := flagSet.Parse(getFlagArgs(ownerConsentCommandsFlagID)); err != nil {
+		logger.ErrorErr(err, "Cannot parse %s flag", ownerConsentCommandsFlagID)
```

Comment:
Should we continue if we can't parse the consent flag? The flag handling would probably needs to be rewritten.

Reply:
I followed the current design, e.g. `domains` parsing also only logs the error and continues with default/file config. Exiting with a proper error seems more reasonable for me too.